aes/src/armv8/encdec.rs

```diff
@@ -18,21 +18,23 @@ pub(super) unsafe fn encrypt<const KEYS: usize>(
 ) {
     assert!(KEYS == 11 || KEYS == 13 || KEYS == 15);
     let (in_ptr, out_ptr) = block.into_raw();
-    let mut block = vld1q_u8(in_ptr.cast());
+    let mut block = unsafe { vld1q_u8(in_ptr.cast()) };
 
     for &key in &keys[..KEYS - 2] {
-        // AES single round encryption
-        block = vaeseq_u8(block, key);
-        // Mix columns
-        block = vaesmcq_u8(block);
+        unsafe {
+            // AES single round encryption
+            block = vaeseq_u8(block, key);
```

Comment:
I think it's better to just enable the lint for such modules than to introduce this amount of `unsafe` blocks.